packages/broker/src/plugins/operator/VoteOnSuspectNodeService.ts

```diff
@@ -1,15 +1,36 @@
 import { Logger } from '@streamr/utils'
+import StreamrClient from 'streamr-client'
+import { OperatorServiceConfig } from './OperatorPlugin'
+import { VoteOnSuspectNodeHelper } from './VoteOnSuspectNodeHelper'
 
 const logger = new Logger(module)
-
 export class VoteOnSuspectNodeService {
-    // eslint-disable-next-line class-methods-use-this
+    private readonly streamrClient: StreamrClient
+    private readonly voteOnSuspectNodeHelper: VoteOnSuspectNodeHelper
+
+    constructor(streamrClient: StreamrClient, serviceConfig: OperatorServiceConfig) {
+        this.streamrClient = streamrClient
+        this.voteOnSuspectNodeHelper = new VoteOnSuspectNodeHelper(
+            serviceConfig,
+            this.handleNodeInspectionRequest.bind(this)
+        )
+    }
+
     async start(): Promise<void> {
-        logger.info('started')
+        logger.debug('starting')
+        await this.voteOnSuspectNodeHelper.start()
+        logger.debug('started')
```

Comment:
A bit redundant this logging maybe since `voteOnSuspectNodeHelper.start()` already contains logging
```suggestion
        await this.voteOnSuspectNodeHelper.start()
```